src/game_objects/state.py

```diff
@@ -161,6 +171,27 @@ def __add_stats_for_item(self, item):
         # Can the .get thing be actually false ?!
         if item_info.guppy:
             self.guppy_set.add(item)
+        # If this can make us beelzebub, check if we're beelzebub
```

Comment:
We can avoid this code duplication by creating a "transformation_list" in `item.py`, just like we used a stat_list, and iterate through it to check if we need to add the item to a specific transformation set.
(this is also true for the others 'if' in the code which are related to transformations)